common/common-shim/pom.xml

```diff
@@ -88,5 +96,37 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro</artifactId>
+      <version>${org.apache.avro.version}</version>
+      <!--<scope>provided</scope>-->
+    </dependency>
+    <dependency>
+      <groupId>com.google.gdata</groupId>
+      <artifactId>core</artifactId>
+      <version>1.47.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.gdata</groupId>
+      <artifactId>gdata-core</artifactId>
+      <version>1.41.4</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>${jackson.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
we shouldn't hardcode such dependency, at least version properties, and also defining in assembly.xml 
https://github.com/pentaho/pentaho-hadoop-shims/pull/529/files#diff-f88d60091735d4b39ab3f1ec0c80218bR21 like here is enough, we should trust what vendor takes. Why is here hardcode?